scripts/helmer.sh

```diff
@@ -0,0 +1,206 @@
+#!/usr/bin/env bash
+set -aeEuo pipefail
+# trap 'echo "Error on ${BASH_SOURCE/$PWD/.}:${LINENO} $(sed -n "${LINENO} s/^\s*//p" $PWD/${BASH_SOURCE/$PWD})"' ERR
+
+. "$(dirname "$0")/../helpers/kubelib.sh"
+
+# ==================================================================================================
+# Global variables & Checks - Used only in install & upgrade actions
+
+NAMESPACE=${NAMESPACE:-kubewarden}
+# Kubewarden helm repository
+REPO_NAME=${REPO_NAME:-kubewarden}
+# Use charts from [./dirname|reponame]
+CHARTS_LOCATION=${CHARTS_LOCATION:-$REPO_NAME}
+# [next|prev|v1.17.0-rc2|local]
+VERSION=${VERSION:-$( [[ "$CHARTS_LOCATION" == */* ]] && echo "local" || echo "next" )}
+
+# Extra parameters for helm install
+CRDS_ARGS="${CRDS_ARGS:-}"
+CONTROLLER_ARGS="${CONTROLLER_ARGS:-}"
+DEFAULTS_ARGS="${DEFAULTS_ARGS:-}"
+# Use latest tag for main images
+if [ -n "${LATEST:-}" ]; then
+    DEFAULTS_ARGS="--set policyServer.image.tag=latest $DEFAULTS_ARGS"
+    CONTROLLER_ARGS="--set image.tag=latest --set auditScanner.image.tag=latest $CONTROLLER_ARGS"
+fi
+
+# Add missing "v" prefix
+[[ $VERSION =~ ^[1-9] ]] && VERSION="v$VERSION"
+
+# Check if local charts are available
+[ "$VERSION" = local ] && test -d "$CHARTS_LOCATION/kubewarden-crds"
+
+# Remove kubewarden- prefix from chart name
+[ $# -gt 1 ] && set -- "$1" "${2/#kubewarden-}" "${@:3}"
+
+# Second parameter must be short chart name or empty
+[[ ${2:-} =~ ^(crds|controller|defaults)?$ ]] || { echo "Bad chart: $2"; exit 1; }
+
+# ==================================================================================================
+# Configuration of helm versions & values
+
+print_env() {
+    # Main parameters
+    echo NAMESPACE=\"$NAMESPACE\"
+    echo REPO_NAME=\"$REPO_NAME\"
+    echo CHARTS_LOCATION=\"$CHARTS_LOCATION\"
+    echo VERSION=\"$VERSION\"
+    # Extra parameters
+    echo CRDS_ARGS=\"$CRDS_ARGS\"
```

Comment:
Quibble: I would prefer if the args would be read from a temporal values.yaml file per chart, and I could select a different path or manually edit between tests. Sometimes while developing a feature I need a specific setup of Kubewarden, and I can provide that with a big chunk of settings in a values.yaml. Or I need to modify a big chunk of the values.yaml to see how the controller behaves, and would like to run smoke tests too or so.

This is not a blocker and should be done if anything on a different card and PR.

Reply:
I think this is doable with combination of helmer calls and ARGS.
For example, you can do:
```bash
# Initially
make cluster install CONTROLLER_ARGS="-f values.yaml"

# Later in tests to change it:
helmer set controller -f other_values.yaml
```